kratos/python_scripts/eigen_solver_factory.py

```diff
@@ -6,10 +6,15 @@ def ConstructSolver(settings):
     if(type(settings) != KratosMultiphysics.Parameters):
         raise Exception("Input is expected to be provided as a Kratos Parameters object")
     
+    solver_type = settings["solver_type"].GetString()
+    
+    if solver_type == "eigen_sparse_eigensystem":
```

Comment:
name inconsistent? see above

Reply:
Why does it work also with different names?

Reply:
i would suggest ```eigen_sparse_eigenvalue_solver```

Also for the class/filename i would suggest to replace "Eigensystem" with "Eigenvalue". Then it would be more consistent with other, already existing ones.
But FEAST for example also does not follow that "convention"...

Reply:
I [choose](http://www.springer.com/de/book/9783540380047) [this](https://www.gnu.org/software/gsl/manual/html_node/Eigensystems.html) [name](http://reference.wolfram.com/language/ref/Eigensystem.html) because it also calculates the eigenvectors. But if there is a convention we should rename it.

Reply:
@oberbichler i agree with that argumentation. Personally i am more familiar with the term eigenvalue problem.
@philbucher is there a convention or are we free to name it how we like?